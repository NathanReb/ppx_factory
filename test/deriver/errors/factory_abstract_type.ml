type t = int
[@@deriving factory]
