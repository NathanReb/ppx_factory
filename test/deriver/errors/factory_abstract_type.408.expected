File "factory_abstract_type.ml", line 1, characters 0-33:
1 | type t = int
2 | [@@deriving factory]
Error: ppx_factory.factory: cannot derive from abstract type. Has to be a record or variant type.
